app/houses/[hid]/page.tsx

```diff
@@ -0,0 +1,53 @@
+import getHouse from "@/app/lib/getHouse";
+import Image from "next/image";
+
+interface Params {
+  params: {
+    hid: string;
+  }
+}
+
+const House = ({ params: { hid }}: Params) => {
+  const house = getHouse(hid);
```

Comment:
this component needs [generateStaticParams](https://nextjs.org/docs/app/api-reference/functions/generate-static-params) and passing the "available urls" so that the page can be statically generated